docs/metrics.md

```diff
@@ -90,8 +90,9 @@ Extra labels:
 * `drive` - Disk device that is being written/read.
 * `type` - Whether it's a read or write operation.
 
-#### kubevirt_vmi_storage_times_ms_total
-#### HELP kubevirt_vmi_storage_times_ms_total storage operation time.
+#### kubevirt_vmi_storage_times_ns_total
+#### HELP kubevirt_vmi_storage_times_ns_total storage operation time. (deprecated in favor of kubevirt_vmi_storage_times_seconds_total)
+#### HELP kubevirt_vmi_storage_times_seconds_total storage operation time.
```

Comment:
I'm having troubles with `make generate` again, this was the result of keeping both metrics. Even though they are indeed two separate metrics. 

Result of hitting the metrics endpoint:
```
# HELP kubevirt_vmi_storage_times_ns_total storage operation time. (deprecated in favor of kubevirt_vmi_storage_times_seconds_total)
# TYPE kubevirt_vmi_storage_times_ns_total counter
kubevirt_vmi_storage_times_ns_total{drive="vda",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="read"} 2.803209064e+09
kubevirt_vmi_storage_times_ns_total{drive="vda",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="write"} 2.042818e+07
kubevirt_vmi_storage_times_ns_total{drive="vdb",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="read"} 6.0506786e+07
kubevirt_vmi_storage_times_ns_total{drive="vdb",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="write"} 0
# HELP kubevirt_vmi_storage_times_seconds_total storage operation time.
# TYPE kubevirt_vmi_storage_times_seconds_total counter
kubevirt_vmi_storage_times_seconds_total{drive="vda",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="read"} 2.803209064
kubevirt_vmi_storage_times_seconds_total{drive="vda",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="write"} 0.02042818
kubevirt_vmi_storage_times_seconds_total{drive="vdb",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="read"} 0.060506786
kubevirt_vmi_storage_times_seconds_total{drive="vdb",kubernetes_vmi_label_kubevirt_io_nodeName="node01",kubernetes_vmi_label_special="vmi-fedora",name="vmi-fedora",namespace="default",node="node01",type="write"} 0
```